builtin/providers/heroku/resource_heroku_space.go

```diff
@@ -128,6 +132,19 @@ func resourceHerokuSpaceDelete(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+func resourceHerokuSpaceImport(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
+	client := m.(*heroku.Service)
+
+	spaceRaw, _, err := SpaceStateRefreshFunc(client, d.Id())()
+	if err != nil {
+		return nil, err
+	}
+	space := spaceRaw.(*heroku.Space)
+
+	setSpaceAttributes(d, space)
+	return []*schema.ResourceData{d}, nil
+}
```

Comment:
@vanstee Since this function is basically identical to `resourceHerokuSpaceRead`, is it possible to use [`schema.ImportStatePassthrough`](https://github.com/hashicorp/terraform/blob/f27aa3c826b02b84c8b5d67efd15479504df08c1/builtin/providers/github/resource_github_repository.go#L18-L20) (example from the github provider linked)?

`schema.ImportStatePassthrough` basically reuses the `Read` function to do the import. As long as the `Read` function only uses `d.Id()` to do its work (and not, for example, `d.Get("something")`), it should work properly.